### DISCUSSION TITLE
Contributions graph problem

### DISCUSSION THREAD

**Discussion by aminos02** [community/community#34703] [Other Feature Feedback, Questions, & Ideas]
If I create a private repository and I push my commits ,and when someone enters my github profile ,the contributions graph doesn't show my contributions.
I tested the same actions in my friend github and it's working fine , So I think I have a problem in my github profile.

**Comment by jfullstackdev** [comment]
you must enable it 

![image](https://user-images.githubusercontent.com/47092464/193273946-9901f201-0ed9-4058-a98f-e8b62f2504fc.png)

**Comment by aminos02** [comment]
Thank you for your response. It works  well.
